preview-site-src/elements/miscellaneous.adoc

```diff
@@ -37,3 +37,12 @@ This is the same in-page cross reference:
 This is an in-page cross reference for the #bold ID: <<bold,Click me>>.
 
 xref:table.adoc[Go to the table page]
+
+== Headings
+
+= H1 Test Heading
+== H2 Test Heading
```

Comment:
I know this is for testing, but every time I run `npm run preview`, this triggers the error message "asciidoctor: ERROR: <stdin>: line 43: level 0 sections can only be used when doctype is book", which sounds scary even though it isn't. Maybe just refer it to the actual heading of this page, which happens to be an H1?

H2 can refer to the H2 headings too, so H3...H6 Test Headings can be nested under `== Headings`.

Reply:
Absolutely fair. Removed the H1 to eliminate the error.